doc/integrator/ngeo.rst.tmpl

```diff
@@ -117,3 +99,14 @@ You also have a ``default`` entry from where we can give the base values for ``c
 The possible values are described in the ngeo documentation
 `ngeo constants <https://camptocamp.github.io/ngeo/${MAIN_BRANCH}/jsdoc/module-src_options.html>`_,
 `gmf constants <https://camptocamp.github.io/ngeo/${MAIN_BRANCH}/jsdoc/module-contribs_gmf_src_options.html>`_.
+
+CSS style
+---------
+
+In the ``gmfOptions`` we have a ``cssVars`` property where we can configure the CSS variable
+(``var(--a-variable)`` in the css files.
+
+On this dictionary we have some particular variables named ``brand-primary`` and ``brand-secondary``,
```

Comment:
This confuses me a bit... I suppose the dictionary is empty, but the integrator can add any variables she wishes (variables which correspond to the CSS variable names). And "brand-primary" and "brand-secondary" are then somehow not normal CSS variables used in GMF, but something hard-coded?

Reply:
I completely rephrase it, I hope that's clearer...